cookbook/email/dev_environment.rst

```diff
@@ -119,6 +119,66 @@ the replaced address, so you can still see who it would have been sent to.
     These are ``X-Swift-Cc`` and ``X-Swift-Bcc`` for the ``CC`` and ``BCC``
     addresses respectively.
 
+Sending to a Specified Address but with Exceptions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Suppose you want to have all email redirected to a specific address,
+(like in the above scenario to ``dev@example.com``).
+But then you may want email sent to some specific email addresses to go through
```

Comment:
this should be joined with the previous line, only add line breaks when you're crossing the 72th character.